src/System.Drawing.Primitives/tests/System.Drawing.Primitives.Tests.csproj

```diff
@@ -7,8 +7,8 @@
     <RootNamespace>System.Drawing.Primitives.Tests</RootNamespace>
     <AssemblyName>System.Drawing.Primitives.Tests</AssemblyName>
     <ProjectGuid>{297A9116-1005-499D-A895-2063D03E4C94}</ProjectGuid>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Even if it isn't necessary today I would suggest adding the condition to help prevent issues tomorrow when we add another configuration.